examples/Custom_Hooks/callable.py

```diff
@@ -0,0 +1,92 @@
+## E2E Test for using callables
+
+# This example shows how to use the register_hook function to request internal data from different objects such as campaign, recommenders and objective.
+
+# To request internal data from the recommendation process this function can be used by giving it two arguments:
+# - The class method that contains the required data
+# - The callable function that will process the data
+# The function wraps both functions and returns the wrapped function
```

Comment:
dont really udnerstand this snetence, its not wrapping both, its wrapping the target and in the process `attaches` or calls the hook